coderd/coderd.go

```diff
@@ -440,6 +440,13 @@ func New(options *Options) *API {
 					CurrentVersion:               buildinfo.Version(),
 					WorkspaceProxiesFetchUpdater: *(options.WorkspaceProxiesFetchUpdater).Load(),
 				},
+				ProvisionerDaemons: healthcheck.ProvisionerDaemonsReportDeps{
+					CurrentVersion:         buildinfo.Version(),
+					CurrentAPIMajorVersion: provisionersdk.CurrentMajor,
+					Store:                  options.Database,
+					TimeNowFn:              dbtime.Now,
+					StaleInterval:          provisionerdserver.DefaultHeartbeatInterval * 3,
```

Comment:
Should we remove those that have defaults now? Otherwise the might be updated in one place but not actually propagated.